src/utils/send-email.js

```diff
@@ -0,0 +1,70 @@
+const AWS = require('aws-sdk');
+const config = require('../config/index');
+const {
+  SUCCEEDED,
+} = require('../api/general-services/pipeline-manage/constants');
+
+const sendEmail = (experimentId, status, user) => {
+  const ses = new AWS.SES({
+    region: config.awsRegion,
+    apiVersion: '2010-12-01',
+  });
+
+  const buildEmailBody = () => {
```

Comment:
Can you refactor out this component (put it out of this `sendEmail` function) and rename it to something that is more specific? (e.g. `buildPipelineStatusEmailBody`). The function should also not rely on global `user`, `status` and `experimentId` and take them in as parameters instead,